uc_model.py

```diff
@@ -148,7 +155,9 @@
 print("model setup complete")
 
 from pyomo.opt import SolverFactory
-cbc = SolverFactory('cbc')
+solver = SolverFactory(solver_name)
 
 print("solving")
-cbc.solve(m, options={'ratioGap':0.01}, tee=True)
+options = {'gurobi':{'ratioGap':0.01}, 'glpk':{'mipgap':0.01}, 'cbc':{'ratioGap':0.01}, 'scip':{'mipgap':0.01}}
```

Comment:
Is `ratioGap` the right option for Gurobi?

Reply:
I changed this to 'mipgap' and confirmed that Gurobi sees this value.